vector/src/main/java/im/vector/fragments/RoomsFragment.java

```diff
@@ -280,7 +280,7 @@ private void refreshRooms() {
                 // test
                 if ((null != room) && // if the room still exists
                         !room.isConferenceUserRoom() && // not a VOIP conference room
-                        !directChatRoomIds.contains(room.getRoomId()) &&
+                        //add direct rooms !directChatRoomIds.contains(room.getRoomId()) &&
```

Comment:
@jp-modernisation-gouv-fr remove the line here. The comment to inform that direct chats are not excluded anymore is useful. It should be kept (displayed before the test block)

Reply:
ok